### PR TITLE
Add JCB as a coming soon to the list of available payment methods

### PR DESCRIPTION
**Pull request by mordeth**
#### Changes proposed in this Pull Request
In order to promote the JCB payment method to Japanese merchants with one of the upcoming releases we added it to the list of available payment methods with a coming soon state.

Screenshot 2023-08-04 at 1 00 41

Screenshot 2023-08-04 at 1 00 48

Screenshot 2023-08-04 at 1 00 30

#### Testing instructions
- Navigate to `Payments` -> `Settings`.
- Scroll down to `Payments accepted on checkout`.
- Verify the JCB payment method is in the list.
- Verify JCB payment method has no checkbox and cannot be activated.
- Hover the icon and verify the tooltip is displayed saying the JCB payment method is coming soon.
- Verify the fees are `3.6% + 0`.

*

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

File: changelog/add-jcb-payemnt-method [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: minor
```
Comment (elazzabi) [inline changelog/add-jcb-payemnt-method]: There is a small typo in the name of this file: `payemnt` => `payment`

**Review by elazzabi — COMMENTED** [review]
Changes looks good. Tested locally with the server branch.

I added some comments to the server branch that will probably require changes to this PR too...

**Comment by elizaan36** [comment]
hey @mordeth Could you share a screenshot of the mobile view. I'm seeing the tooltip get cut off in other instances. Shared in `paJDYF-9Mc`

Also shared in that post that I'd like to make a transition to the white background for tooltips but we'll create a new issue for that.

**Comment by elazzabi** [comment]
@elizaan36 👋 

This is how it looks on mobile for me:

Screenshot 2023-08-03 at 17 07 06

**Comment by elizaan36** [comment]
Thanks, and how does the tooltip appear to you?

**Review by mordeth — COMMENTED** [review]
@elizaan36 The left part of the tooltip is cut for me also. However, the code doesn't take into account it is mobile version and always sets `left` by getting the middle of the element minus the tooltip width / 2. I think if we center the icon element on mobile it would resolve the problem.

Screenshot 2023-08-03 at 20 08 32

**Comment by elizaan36** [comment]
Thanks for sharing the mobile screenshot. A few things:

- The tooltip component has a variation with a left arrow for these use cases. We can use the different tooltip variation rather than moving elements in the UI. As part of this issue, if you could make this update holistically to all tooltips that appear on payments settings, it would fix a major accessibility issue. The tooltip should adapt to the viewport.
- The opacity of the row is still 100%, whereas in other examples the payment method row is grayed out to indicate inactivity. Could we align with the approach taken for this component elsewhere?
- The tooltip is not aligned with the icon, the arrow should appear centered with the icon (either vertically or horizontally). 
image

File: changelog/add-jcb-payemnt-method (hunk 1 above)
Comment (mordeth) [reply changelog/add-jcb-payemnt-method]: Thank you, resolved!

**Review by elazzabi — APPROVED** [review]
Looks good to me after the changes 👍

**Comment by elazzabi** [comment]
@elizaan36 should we move those notes to a separate issue so we can work on them? I believe some of these are out of the scope of this PR, but we should definitely fix them.

**Review by dpaun1985 — APPROVED** [review]
✅ LGTM

**Review by anu-rock — APPROVED** [review]
Great work here 👏 

___

#### Review checklist

| Parameter  | Result |
| ------------- | ------------- |
| Approach  | ✅ All good |
| Implementation  | ✅ All good |
| Automated tests  | Not applicable |
| Naming  | ✅ All good |
| Typing  | ✅ All good |
| Code comments  | ✅ All good |
| Changelog  | ✅ All good |
| Testing instructions  | ✅ All good |
| Manual testing  | ✅ Works as expected |

#### Scenarios tested

| Scenario  | Result |
| ------------- | ------------- |
| JCB is shown in the list of payment methods  | ✅ |
| JCB list item is greyed out  | ✅ |
| Icon to indicate that this feature is coming soon  | ✅ |
| Fees correctly visible in chip and tooltip  | ✅ |

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `6905` or branch name `add/jcb-payemnt-method` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&branches.woocommerce-payments=add/jcb-payemnt-method) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: c4f74682f0b8ec5235eb4b9c92a91e812e02eac8
- Build time: 2023-08-04 11:00:46 UTC

_Note: the build is updated when a new commit is pushed to this PR._

**Comment by elizaan36** [comment]
@elazzabi Yep I commented somewhere else that we'll need to fix the tooltip accessibility bug in a new issue.

**Pull request merged**